docker-compose.yml

```
@@ -36,6 +36,33 @@ services:
     ports:
       - 8080:8080
 
+  admin:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile.admin
+    restart: always
+    depends_on:
+      - shared
+      - postgres
+    volumes:
+      - ./backend/src:/usr/src/app/backend/src
+      - ./shared/src:/usr/src/app/shared/src
+      - shared:/usr/src/app/shared/dist
+    env_file:
+      - backend/.env.docker
+    environment:
+      - ADMIN_PORT=8081
+      - ADMIN_DISABLE_AUTH=true
```

Shouldn't the environment be updated also to have ADMIN_COOKIE_SECRET?
```suggestion
    environment:
      - ADMIN_PORT=8081
      - ADMIN_DISABLE_AUTH=true
      - ADMIN_COOKIE_SECRET=adminCookieSecret
```

---

Same as below
https://github.com/twoje-osk/twoje-osk/pull/49#discussion_r928139549